tensorflow/compiler/xla/window_util.cc

```diff
@@ -185,6 +185,17 @@ bool HasWindowReversal(const Window& window) {
   return false;
 }
 
+bool AllOrNoneReversed(const Window& window) {
+  if (window.dimensions().size() == 0) {
+    return true;
+  }
+  bool reversed = window.dimensions()[0].window_reversal();
+  return std::all_of(window.dimensions().begin(), window.dimensions().end(),
```

Comment:
Nit, would prefer `absl::c_all_of` if that's possible?  Lives in `absl/algorithm/container.h`.  Otherwise LGTM.

Reply:
Sure, I just figured I'd align it with the rest of this file, which all use `std::all_of`. Alternative, I can push a prep commit to this PR that switches those over also.

Reply:
Oh, I didn't look at the rest of the file!  As you have it is fine.  I thought I changed all of them to c_foo at one point, but maybe some new ones snuck in!

I'd certainly take the cleanup patch, but it's also easy for me to do it internally.

Reply:
Yeah, why don't you do it internally. Getting PRs through always takes a few rounds through CI.